admiral/common_functions.sqf

```diff
@@ -37,25 +37,28 @@ adm_common_fnc_placeMan = {
 };
 
 adm_common_fnc_placeVehicle = {
-    FUN_ARGS_3(_vehicleClassNames,_area,_position);
+    params ["_vehicleClassNames","_area","_position"];
 
-    private ["_classNameData", "_className", "_classNameArguments", "_vehiclePosition", "_vehicle"];
-    _classNameData = SELECT_RAND(_vehicleClassNames);
-    _classNameArguments = [];
+    private _classNameData = SELECT_RAND(_vehicleClassNames);
+    private _classNameArguments = [];
     if (typeName _classNameData == "ARRAY") then {
-        _className = _classNameData select 0;
+        private _className = _classNameData #0;
```

Comment:
This won't work, `_className` is used outside the scope of `if` and has to be declared where it was originally.

Reply:
Yeap, just realised this in testing.